github/orgs_teams_test.go

```diff
@@ -499,3 +499,25 @@ func TestOrganizationsService_ListUserTeams(t *testing.T) {
 		t.Errorf("Organizations.ListUserTeams returned %+v, want %+v", teams, want)
 	}
 }
+
+func TestOrganizationsService_ListPendingTeamInvitations(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/teams/1/invitations", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testFormValues(t, r, values{"page": "1"})
```

Comment:
Please add a test for the custom header:

`testHeader(t, r, "Accept", mediaTypeOrgMembershipPreview)`